lib/taurus/core/tango/tangoattribute.py

```diff
@@ -448,8 +449,17 @@ def poll(self, **kwargs):
             if single:
                 self.read(cache=False)
             else:
-                self.__attr_value = self.decode(kwargs.get('value'))
+                value = self.decode(kwargs.get('value'))
                 self.__attr_err = kwargs.get('error')
+                filter_old_event = getattr(tauruscustomsettings,
+                                           'FILTER_OLD_TANGO_EVENTS', False)
+                if self.__attr_value is not None and \
```

Comment:
Also, this filter block should be properly documented, since its justification is not obvious